catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorTest.java

```diff
@@ -71,17 +75,19 @@ public void numericValuesTest() {
 				.perform(click());
 
 		//typeText not working for formula editor, so use CustomActions.typeInValue
-		onFormulaEditor()
-				.performEnterNumber(12345.678);
+		onFormulaEditor().performEnterNumber(12345.678);
 		pressBack();
 	}
 
 	@After
 	public void tearDown() throws Exception {
+		//CatroidKoinHelperKt.stop(dependencyModules);
```

Comment:
??

Reply:
@juliajulie95 Well spotted! Interesting that these don't get reported at the Static Analysis level.